src/goes/test_goes_converter.py

```diff
@@ -0,0 +1,33 @@
+#
+# test_goes_converter.py
+#
+# This script is a driver for the GoesConverter class.
+#
+#
+import os
+import time
+
+from solo.basic_files import tree
+from goes_converter import GoesConverter
+
+
+def test_goes_converter():
+    input_files_path = "/Users/eric2/PROJECTS/GOES16_TEST_FILES/input/12_15_2020/00"
```

Comment:
I suspect that the files you are using for this test are large and inappropriate for unit testing (runtime is too long, ie longer than a few seconds). If that's the case, then I suggest making these variables, with the hardcoded paths, arguments to this script so that the hardcoded paths can be eliminated.

If these paths are pointing to small data that runs quickly, then please turn this into a "ctest" style test. You can look in one of the other source files to see how that is done. I can help with this too.

Thanks!

Reply:
Yes. The process to convert 16 channels into the two IODAv2 files takes around 3 minutes. This Python script is really just a driver at this point. I will modify this script to be a proper "ctest" style test following guidance from you and the other source files. Thank you for this input.

Reply:
3 minutes is a bit too long for a unit test. If it's simple and easy to trim the input files down so that the test runs in a couple seconds, then it may be worthwhile to do that for this PR. If it's going to take some work to create a small, fast test then I think it might be better to get rid of the hard coded paths (ie, add arguments to this script) and call it good for this PR. It would be okay to add the unit test in a subsequent PR.

Reply:
I will work on configuring a quick test in the next PR. I pushed a commit to remove the hard-coded paths for this test driver script.